wsdl/salesforce-outbound-message.wsdl

```diff
@@ -0,0 +1,137 @@
+            <import namespace="urn:enterprise.soap.sforce.com" />
+            <import namespace="urn:sobject.enterprise.soap.sforce.com" />
+    
+            <element name="notifications">
+                <complexType> 
+                    <sequence> 
+                    	<element name="OrganizationId" type="ent:ID" />
+                    	<element name="ActionId" type="ent:ID" />
+                    	<element name="SessionId" type="xsd:string" nillable="true" />
+                    	<element name="EnterpriseUrl" type="xsd:string" />
+                    	<element name="PartnerUrl" type="xsd:string" />
+                        <element name="Notification" maxOccurs="100" type="tns:ContactNotification" />
+                    </sequence> 
+                </complexType> 
+            </element>
+    
+            <complexType name="ContactNotification">
+                <sequence>
+                    <element name="Id" type="ent:ID" />
+                    <element name="sObject" type="ens:Contact" />
+                </sequence>
+            </complexType>
+                
+            <element name="notificationsResponse">
+                <complexType>
+                    <sequence>
+                        <element name="Ack" type="xsd:boolean" />
+                    </sequence>
+                </complexType>
+            </element>
+        </schema>
+    </types>
+    
+    
+    <!-- Method Messages -->   
+    <message name="notificationsRequest">
+        <part element="tns:notifications" name="request"/>
+    </message>
+    <message name="notificationsResponse">
+        <part element="tns:notificationsResponse" name="response"/>
+    </message>
+
+    <!-- PortType -->
+    <portType name="NotificationPort">
+        <operation name="notifications">
+            <documentation>Process a number of notifications.</documentation>
+            <input  message="tns:notificationsRequest"/>
+            <output message="tns:notificationsResponse"/>
+        </operation>
+    </portType>
+    
+    <!-- Binding 
+         You need to write a service that implements this binding to receive the notifications
+     -->
+    <binding name="NotificationBinding" type="tns:NotificationPort">
+        <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+        
+        <operation name="notifications">
+            <soap:operation soapAction=""/>
+            <input>
+                <soap:body use="literal"/>
+            </input>
+            <output> 
+                <soap:body use="literal"/>
+            </output>
+        </operation>
+    </binding>
+
+    <!-- Service Endpoint -->
+    <service name="NotificationService">
+        <documentation>Notification Service Implementation</documentation>
+        <port binding="tns:NotificationBinding" name="Notification">
+            <soap:address location="https://your-url/endpoint"/>
```

Comment:
if we ever update this with a new wsdl from salesforce we have to make sure to edit this since it will write the real url here that might contain secrets.